src/main/kotlin/net/atos/zac/admin/ZaakafhandelParameterBeheerService.kt

```diff
@@ -227,12 +231,12 @@ class ZaakafhandelParameterBeheerService @Inject constructor(
         val nieuweResultaattypen = nieuwZaaktype.resultaattypen.map { ztcClientService.readResultaattype(it) }
         nieuweZaakafhandelParameters.nietOntvankelijkResultaattype =
             vorigeZaakafhandelparameters.nietOntvankelijkResultaattype?.let {
-                mapVorigResultaattypeOpNieuwResultaattype(nieuweResultaattypen, it)
+                mapVorigResultaattypeOpNieuwResultaattype(it, nieuweResultaattypen)
```

Comment:
Why this change? Was the original code wrong? How does this relate to this PR?

Reply:
to align all the method calls in this service --> first param is old, next one is new